36/generated/connect_rest.yaml

```diff
@@ -8,7 +8,7 @@ info:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
   title: Kafka Connect REST API
-  version: 3.6.1
+  version: 3.6.2-SNAPSHOT
```

Comment:
shouldn't be snapshot

Reply:
Again we don't want this change. The docs should cover the last released version for 3.6, hence 3.6.1.